understanding/22/redundant-entry.html

```diff
@@ -51,6 +51,8 @@ <h2>Intent of Redundant Entry</h2>
             <li>tick a checkbox to populate inputs with the same values as previously entered (e.g., my billing address is the same as my shipping address).</li>
         </ul>
 
+        <p>The location of the data "available to select" is evaluated by the Web page, so it needs to be on the same page. Ideally, it would be visible by default and closely associated with the input where the data is required. However, it could be elsewhere on a page, including within a show/hide component.</p>
```

Comment:
```suggestion
        <p>Data which is "available to select" would need to be on the same page. Ideally, it would be visible by default and closely associated with the input where the data is required. However, it could be elsewhere on a page, including within a show/hide component.</p>
```